src/views/asset-view/layout.tsx

```diff
@@ -56,6 +61,14 @@ const AssetSideBar = ({
             assetType={assetType}
             assetReference={assetId}
           />
+          <details className="govuk-details" data-module="govuk-details">
+            <summary className="govuk-details__summary">
+              <span className="govuk-details__summary-text">Property Specification</span>
+            </summary>
+            <div className="govuk-details__text">
+              <PropertySpecification assetCharacteristics={assetCharacteristics} />
+            </div>
+          </details>
```

Comment:
This looks like something that ought to belong within the new component, or am I mistaken?